.github/workflows/ci.yml

```diff
@@ -87,10 +100,23 @@ jobs:
       # Action for this, as of Jan 2021 it does not support Python coverage
       # files - only lcov files.  Therefore, we use coveralls-python, the
       # coveralls.io-supplied Python reporter, for this.
-      - name: "Report Coverage to Coveralls"
+      #
+      # If we `pip install coveralls` (with Python 2.7), we will get
+      # coveralls-python 1.11.1.  If we `pip3 install coveralls` (with
+      # Python 3), we will get coveralls-python 3.0.1.
```

Comment:
This comment will be stale fairly soon, I think.  Either pin the version or leave out the details about exactly which version results.

Reply:
Only coveralls-python 1.x branch has maintained Python 2 compatibility, but that is incompatible with 3.x branch.  Will pin and update comments here.

Reply:
Also, coveralls-python 1.11.1 does not have `--finish`, so we can't downgrade uniformly.